src/skottie-player.ts

```diff
@@ -352,6 +352,37 @@ export class SkottiePlayer extends LitElement {
     }
   }
 
+  /**
+   * Renders text inputs to change text assets in the animation.
+   */
+  renderTextAssets(): TemplateResult | string {
+    if (this.content.assets) {
+      return html`
```

Comment:
This is a little hard to read maybe get rid of the nested html'' templates, only the last html'' template is needed.